examples/99-advanced/anatomical_groups.py

```diff
@@ -56,14 +56,11 @@
 #     This mapping is specific to the PyVista datasets and is not part of the
 #     TotalSegmentator data.
 
-print(
-    '{\n'
-    + '\n'.join(
-        f'    {"'" + name + "':":<32} ({R:>3}, {G:>3}, {B:>3}),'
-        for name, (R, G, B) in names_to_colors.items()
-    )
-    + '\n}',
-)
+print('{')
+for name, (R, G, B) in names_to_colors.items():
+    key = f"'{name}'"
+    print(f'{key:>32}: ({R:>3}, {G:>3}, {B:>3}),')
```

Comment:
Thanks, this is much better. This may be a style thing but I find it hard to read the label names with the strings right-aligned
```suggestion
    print(f'{key:<32}: ({R:>3}, {G:>3}, {B:>3}),')
```